components/dashboard/farmer/Statistics.tsx

```diff
@@ -27,7 +27,7 @@ async function getStatistics() {
   });
 
   if (data.length == 0) return undefined;
-  const eggs = data.map(datum => (datum.fieldSmallEggs || 0) + (datum.fieldMediumEggs || 0) + (datum.fieldLargeEggs || 0))
+  const eggs = data.map(datum => datum.fieldSmallEggs + datum.fieldMediumEggs + datum.fieldLargeEggs)
```

Comment:
Just double checking: do we know for sure these properties are going to be `int`'s?

Reply:
with my defined schema and client validation, yes